lib/Default/AbstractSmrPlayer.class.php

```diff
@@ -542,6 +544,17 @@ public function isNPC() {
 	public function hasNewbieStatus() {
 		return $this->newbieStatus;
 	}
+	
+	/**
+	 * Does the player have newbie status and
+	 * a low attack rating?
+	 */
+	public function isNoncombatNewbie() {
```

Comment:
Since every call to this function also asks if the killer is not in newbie status, I think it would help consistency if this function (renamed, perhaps) took a `SmrPlayer $killer` and included that logic, with a comment explaining that newbies are exempt from the penalties to killing other newbies.